libtiledbvcf/src/stats/variant_stats.cc

```diff
@@ -435,7 +456,7 @@ void VariantStats::process(
       return;
     }
   } else {
-    return;  // ngt <= 0
+    return;  // ngt >= 0
```

Comment:
Let's remove the comment. The return actually happens when `ngt < 0`, which is easy enough to see from the code.

Reply:
whoops; I misinterpreted that as one of the clang-format generated end bracket comments